android/src/main/cpp/LayoutAnimations.cpp

```diff
@@ -100,10 +101,22 @@ void LayoutAnimations::setFindPrecedingViewTagForTransition(
       findPrecedingViewTagForTransitionBlock;
 }
 
+void LayoutAnimations::setGetSharedGroupBlock(
+    GetSharedGroupBlock getSharedGroupBlock) {
+  getSharedGroupBlock_ = getSharedGroupBlock;
+}
+
 int LayoutAnimations::findPrecedingViewTagForTransition(int tag) {
   return findPrecedingViewTagForTransitionBlock_(tag);
 }
 
+jni::local_ref<JArrayInt> LayoutAnimations::getSharedGroup(int tag) {
```

Comment:
This is a nitpick but we can add `const` here.
```suggestion
jni::local_ref<JArrayInt> LayoutAnimations::getSharedGroup(const int tag) {
```